### ISSUE TITLE
Clarifications - api domain, offers, prices, stocks, delivery

### ISSUE THREAD

**Issue by andus4n** [allegro/allegro-api#10035]
Hi...i have a few more things that i'd like to clarify regarding this api:
1. is there anything like api.allegro.**cz** / api.allegro.**sk** / api.allegru.**hu** or all of the calls are made to api.allegro.pl? (except the authorization and sandbox ones, of course)
2. is there any way to retrieve the best selling price amongst all of the offers for a product?
3. how are the delivery days computed, to be more precise? i mean, should all of the days be taken into account or should i exclude the non-working ones?
4. is there any way to change a product's RRP? (RRP = recommended retail price = the price that's "cutted" / appears with a line-through)
5. what's the difference between **$.productSet[0].quantity.value** and **$.stock.available**? why is the first one required to have a value greater than 0?
6. is there any way to list a different stock on the "extra" marketplaces?
7. can i use other values for the delivery time besides the documented ones? (PT0S, PT24H, P2D, P3D, P4D, P5D, P7D, P10D, P14D, P21D, P30D, P60D)
8. how can i retrieve the commission (the amount retained by allegro) for a product?
9. do the delivery days represent the time until a seller dispatches a package to the courier or until the client itself retrieves the package?

That would be all for the moment.

**Comment by MaciejFrackowiak** [comment]
1. No, there is one API available on api.allegro.pl.
2. No, we don't have such an endpoint.
3. This question is not directly connected to the API, please [contact](https://allegro.pl/pomoc/kontakt) Allegro support.
4. No.
5. productSet[0].quantity.value refers to the product part of the request and the value must be greater than 0 because you cannot create productSet with 0. stock.available refers to the offer quantity, which is available in the offer and can be buy by the buyers.
6. No, there is no such possibility. For all the marketplaces stock is the same.
7. No.
8. You can calculate it by [POST /pricing/offer-fee-preview](https://developer.allegro.pl/documentation/#operation/calculateFeePreviewUsingPOST).
9. Similar to point 3, please contact Allegro support.

**Comment by andus4n** [comment]
thank you for the clarifications...
i'd have another thing that i'd need to be enlighten on: i'm using a system to synchronize products by GTIN for the *.pl marketplace...now the question: **_are all of the offers created on *.pl also eligible for the other marketplaces_**?
basically, is it enough to fill the $.additionalMarketplaces object properly in order to create listings on *.hu, *.cz etc? if not, what's the proper way to achieve this?

thank you!

**Comment by AureliuszBrussy** [comment]
Yes, you can list offers only on one marketplace, and they will be displayed on other marketplaces. But, we display offers on other marketplaces when they meet specific conditions. Tou can read in [our guide](https://developer.allegro.pl/tutorials/listing-and-managing-offers-on-foreign-marketplaces-7GndGjeAATn#conditions-for-offers-to-be-displayed-on-additional-marketplaces) about these conditions. 

Baiscally, the delivery method assigned to a given delivery country will determine whether an offer will be qualified for visibility on another B2C service. For example, if in a given offer on allegro.pl, the seller provides or will provide a delivery method to the Czech Republic, such offer will be qualified for visibility also on allegro.cz. 

Of course, you should also determine the price for a given marketplace. You can also use [pricing rules](https://developer.allegro.pl/tutorials/listing-and-managing-offers-on-foreign-marketplaces-7GndGjeAATn#pricing-rules).

**Comment by andus4n** [comment]
thank you very much for the clear and concise responses...
**and now something about the images**: when i'm creating an offer i noticed that allegro doesn't always set the proper "main" (thumbnail) image for me. case and point: i'm creating an offer for a pc game and allegro selects an in-game screenshot as the main image for me...why is this and is there any way to avoid the phenomenon?
basically what i'm askin' is if i could identify the main image from the product search response (`GET /sale/products`).
otherwise, i guess i should just use one of my local images, but wouldn't this increase the waiting time for the offer approval?

thx.

**Comment by MaciejFrackowiak** [comment]
No, it wouldn't take any additional time to approve. You can upload your own images and decide whether to use images from our product catalog, which are assigned to each product. If you don't want to use them, provide [] in the field productSet.product.images.

**Comment by lukeroot** [comment]
Hi @MaciejFrackowiak, Does point 4, 
_Q: is there any way to change a product's RRP? (RRP = recommended retail price = the price that's "cutted" / appears with a line-through)_ _A: No_
Still apply even if you are the verified official seller of the product? 

If this is not the case and we can change it, we are wondering if you action this via the regular `sale/product-offers/` API?

**Comment by MaciejFrackowiak** [comment]
@lukeroot No, you cannot change it.
